### DISCUSSION TITLE
如何挂载百度网盘下的某一个子目录？

### DISCUSSION THREAD

**Discussion by star2k** [AlistGo/alist#1578] [Q&A]
![photo_2022-09-01_09-32-21](https://user-images.githubusercontent.com/9582078/187814234-f4efd46f-05cf-4b7a-9a3d-c4a0a22a25b4.jpg)
![photo_2022-09-01_09-19-17](https://user-images.githubusercontent.com/9582078/187814238-2d7555e0-f928-403d-9d9b-5964527d3186.jpg)
目前挂载根目录是正常的，子目录，如上图，还是显示根目录，且内部文件无法打开，请教挂载子目录的格式？

**Comment by rakaki** [comment]
别的网盘比如阿里是要用浏览器看这个网盘的根目录路径，百度网盘不清楚，主要作者也没写

**Reply by wzmtgggg** [reply]
搞好了吗

**Reply by fyy776735323-cloud** [reply]
直接写 /我的资源 ,提示成功了

**Comment by wzmtgggg** [comment]
搞好了吗

**Comment by miaoxiaozhao** [comment]
没有用，无论怎么设置，都是根目录/
这应该是个bug，请修复

**Reply by LunaMorbus** [reply]
我也是，求解

**Comment by huanleshashou** [comment]
我昨天一开始也是不行，不知道为什么突然就可以了，我感觉是设置完以后没有及时变，等一会看可能就好了

**Comment by haoran37** [comment]
我这解决了，忽略开头的`我的网盘`，直接从`\我的资源`开始，保存后需要到主页右下角的菜单中点下刷新

**Comment by YUAXI** [comment]
直接 /我的资源 就行，之后需要重启一下Alist否则还是显示的根目录下的所有资源
